tests/fixtures/folder_fixtures.py

```diff
@@ -26,6 +26,10 @@ def local_folder(tmp_path):
 def cluster_folder(ondemand_cpu_cluster, local_folder):
     return local_folder.to(system=ondemand_cpu_cluster)
 
+@pytest.fixture
+def local_folder_docker(docker_cluster_pk_ssh, local_folder):
+    return local_folder.to(system=docker_cluster_pk_ssh)
```

Comment:
I think this should be `docker_cluster_pk_ssh_no_auth` for the most basic docker fixture. Looking at the fixtures used in the other tests, `docker_cluster_pk_ssh` is just the base docker cluster fixture used to define the other fixtures, but isn't actually explicitly used in testing